source/slang/core.meta.slang

```diff
@@ -1095,17 +1095,32 @@ struct ConstRef
 typealias __Addr<T> = Ptr<T, $( (uint64_t)AddressSpace::Generic)ULL>;
 
 //@public:
+
+/// `Optional<T>` is a type with one extra value than `T`, this extra value is
+/// used to represent a "missing" or "invalid" `T`. This extra value is called
+/// `none`. 
+///
+/// `none` can be compared against with the operators `==` or `!=`.
+///
+/// An `Optional<T>` value can also be implicitly or explicitly constructed from
+/// a value of type `T`
+///
+/// `Optional<T>` values can be deconstructed with `if(let myT = myOptionalT) ...` 
```

Comment:
@csyonghe , It seems like one back-tick character may need CSS setting.
Currently it is wrapped just with `<span>` without a class name or id.
https://github.com/shader-slang/stdlib-reference/blob/main/types/optional-0/index.md